### PR TITLE
Fix incorrect types in readme

### PR DESCRIPTION
**Pull request by lil-j**
`startPosition` should be a number, not a string.

### CODE REVIEW

**Comment by apal21** [comment]
Hey, thanks for the PR.

Just wanted to know one thing: Is this causing any issue if we use string? Haven't checked yet.

**Comment by lil-j** [comment]
It still works, but by putting a number instead of a string the progress loads more accurately to the requests and progress of the load, overall much smoother IMO. Definitely give it a try and let me know if it does the same for you 😄

**Comment by apal21** [comment]
Okay, I checked the code of Nprogress. The string would work just fine without any problem because of these lines:
https://github.com/rstacruz/nprogress/blob/master/nprogress.js#L325-L326

In Javascript, we can compare a number of type string (eg. `"123"`) with a normal number (`123`).
That means `"0.3" < 1 // returns true` and `"0.3" > 1 // returns false`.

So this cannot have any performance issues.
Still, I'm not closing this PR. Let me know if you find anything else which I couldn't think of or it can cause any problems. We can discuss it. 😇

**Comment by lil-j** [comment]
Hi. This may be true but I have created two websites, identical except one uses a string, and the other uses a number. Notice how the one with a string does not show the progress bar until after the page is loaded while the one with a number begins as it is the request begins, showing a more accurate and true line of progress.
[](https://nextprogresswithstring.vercel.app/)
[](https://nextprogresswithnumber.vercel.app/)

**Comment by lil-j** [comment]
[Here](https://github.com/lil-j/nextprogresswithnumberexample) is the code if you are interested.

**Comment by apal21** [comment]
Insightful!!!
Give me some time. I want to do some research on this. I'll get back to this soon. Thanks for creating this!

**Comment by lil-j** [comment]
No problem man. I just fixed some typos on the site to make some of the wording more clear as well. Keep me posted on your findings!

**Comment by lil-j** [comment]
Hey any updates?

**Comment by apal21** [comment]
Apologies... Last time I couldn't get time. I'll be working to resolve this on this weekend. Let me know if you find anything.

**Comment by apal21** [comment]
Hey @lil-j, I saw the same thing happening with `stopDelayMs`. Can you please change that also and update the title of this PR? I'll merge it.

```javascript
<NextNprogress
  color="#29D"
  startPosition={0.3}
  stopDelayMs={200}
  height="3"
/>

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ And for rendering add `<NextNProgress />` inside `Container` component.
 ```
 <NextNprogress
   color="#29D"
-  startPosition="0.3"
+  startPosition={0.3}
   stopDelayMs="200"
```
Comment (apal21) [inline README.md:23]: For me, `stopDelayMs` is also causing the same effect if I use string. The progress bar shows up only after page load. We can also make this a number.

Comment (lil-j) [reply README.md:23]: Sounds good- will update the request now

**Pull request merged**